roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -24,8 +24,11 @@ def capsule_lce_args(action, capsule_id, env)
   "--csv capsule content #{action}-lifecycle-environment --id #{capsule_id} --lifecycle-environment-id #{env}"
 end
 
+# the correct way is to look at hosts with the Pulpcore feature and the mirror capability
+# but hammer doesn't support that
+internal_capsule = ARGV[0]
```

Comment:
I was thinking this might be `internal_capsule = Socket.gethostname` since the hostname change has already been performed when this script is run.

But your approach of relying on the ansible variable value seems very nice and consistent. :+1:

Reply:
I have bad experience with Python's `socket.getfqdn()`, so I'll pass ;)